<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditStudBook</class>
 <widget class="QDialog" name="EditStudBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="editName"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Автор</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="auth"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Год выпуска</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="yearPubl"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Уч. заведение</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="studHome"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Год обучения</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="yearStud"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Объем страниц</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pages"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Стоимость</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="price"/>
   </item>
   <item>
    <widget class="QPushButton" name="save">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="back">
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
